src/darcyai/pipeline.py

```diff
@@ -757,7 +801,15 @@ def run(self) -> None:
                     self.__logger.exception("Error running Pipeline")
                     if self.__input_stream_error_handler_callback is not None:
                         self.__input_stream_error_handler_callback(e)
+                        try:
+                            reporter.on_pipeline_error(e, False)
+                        except:
+                            pass
```

Comment:
These try..except blocks are a little confusing... May be move all the reporter stuff to methods that ignore exceptions and call those methods here in the `pipeline.run`